DEVELOPMENT.md

```diff
@@ -145,6 +145,7 @@ kustomize build overlays/dev-openshift-imagestream --load_restrictor=LoadRestric
 The dashboard can be accessed by running `kubectl proxy`, which proxies requests through the API Server. You can access the web UI at `localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/`
 
 Important note: the setting for a CSRF secure cookie must be *false* if you intend to access the Dashboard pod through port-forward. Otherwise, it should be true (for example, when using a Route, or Ingress, which is secured with TLS).
+You can disable CSRF secure cookie by adding `--csrf-secure-cookie=false` in the command line args when starting the dashboard backend.
```

Comment:
'You can disable the Secure attribute on the CSRF cookie...'

To make sure it's clear we're not disabling the cookie, just changing one of its attributes.

Reply:
fixed